test/config.json

```diff
@@ -94,7 +94,7 @@
 				"tools/check_make_parser.sh"
 			],
 			"Manual": false,
-			"Shard": 4,
+			"Shard": 5,
```

Comment:
This basically means that this test will be no longer run by travis. Shard 5 is not use by travis tests.

Reply:
That's the intention since it is being run in a GitHub action test. But by keeping it as shard 5, it will still run in a full test run locally.

Reply:
Right. Sorry, my comment was more a note to future self to remember how Travis interacts with the shards.